src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  width: $card-width;
+  box-sizing: border-box;
+  background-color: $card-bg-color;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin-inline: 19px;
+    margin-top: 32px;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+  }
+
+  &__name {
+    color: $text-color-main;
+    margin-block: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    &:hover {
+      color: $card-hover-title;
+    }
+  }
+
+  &__code {
+    display: block;
+    color: $text-color-secondary;
+    margin-top: 4px;
+  }
+
+  &__score {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review {
+    color: $text-color-main;
+  }
+
+  &__price {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      color: $text-color-secondary;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $text-color-main;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    margin-top: 16px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 100%;
+
+    color: $button-color-hover;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $main-blue-color;
+    border-radius: 5px;
+    box-sizing: border-box;
```

Comment:
it is a common practice to apply `box-sizing: border-box` for all elements using `*` selector